From b68c0218628904e0a1160c4720fbc1ee6f43ba02 Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Tue, 26 Nov 2024 23:15:40 +0100
Subject: [PATCH] [doged] Rebrand to Dogecash to doged (#86)

Part of an ongoing rebrand effort. See #82.

Current-ABC adaptation (2026-10-14): Use final lineage doged-node CMake project, isolated installer directory and Doge-Qt bundle/settings identity, retaining version 0.34.1 from the exact new ABC base. Update issue URL to the actual doged-io repository; preserve original copyright holders. Packaging and Qt runtime remain separately unvalidated.

Original-Commit: 34ab40c4a0babb49ec0f85259ee544220a0427a6

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5a4cf2aa92495379568b6186382de7424a576d7f..e322b32a9090ac37fac9812b36e346125ef08668 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,10 +6,10 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
 	"${CMAKE_SOURCE_DIR}/cmake/modules/OverrideInitFlags.cmake"
 )
 
-project(bitcoin-abc
+project(doged-node
 	VERSION 0.34.1
 	DESCRIPTION "doged is a full node implementation of the Dogecoin protocol."
-	HOMEPAGE_URL "https://www.bitcoinabc.org"
+	HOMEPAGE_URL "https://www.doged.io"
 )
 
 add_custom_target(print-version
@@ -24,7 +24,7 @@ add_custom_target(print-project-name
 
 # Package information
 set(PACKAGE_NAME "doged")
-set(PACKAGE_BUGREPORT "https://github.com/Bitcoin-ABC/bitcoin-abc/issues")
+set(PACKAGE_BUGREPORT "https://github.com/doged-io/doged/issues")
 
 # Copyright
 set(COPYRIGHT_YEAR 2026)
diff --git a/PackageOptions.cmake b/PackageOptions.cmake
index a1df82c250858a5b866aadb6f9b7e133c009c21c..e9f5bf0db871cbd97ab545fe12dd796fc88d516f 100644
--- a/PackageOptions.cmake
+++ b/PackageOptions.cmake
@@ -2,9 +2,9 @@
 set(CPACK_PACKAGE_VENDOR "${COPYRIGHT_HOLDERS_FINAL}")
 set(CPACK_PACKAGE_DESCRIPTION "${PROJECT_DESCRIPTION}")
 set(CPACK_PACKAGE_HOMEPAGE_URL "${PROJECT_HOMEPAGE_URL}")
-set(CPACK_PACKAGE_CONTACT "info@bitcoinabc.org")
+set(CPACK_PACKAGE_CONTACT "tobias.ruck@be.cash")
 
-set(CPACK_PACKAGE_INSTALL_DIRECTORY "Bitcoin-abc")
+set(CPACK_PACKAGE_INSTALL_DIRECTORY "doged")
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_SOURCE_DIR}/COPYING")
 set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_SOURCE_DIR}/doc/README_windows.txt")
 
diff --git a/src/qt/CMakeLists.txt b/src/qt/CMakeLists.txt
index 1f82a65e0182cd09d494c9fbea54af431c34ebc6..3c6de67232ac29098bb67eddbd9ce20c97b182a2 100644
--- a/src/qt/CMakeLists.txt
+++ b/src/qt/CMakeLists.txt
@@ -407,18 +407,18 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 		"${EMPTY_LPROJ_FILE}"
 	)
 
-	set(BITCOIN_QT_OSX_BUNDLE_NAME "BitcoinABC-Qt")
+	set(BITCOIN_QT_OSX_BUNDLE_NAME "Doge-Qt")
 	set_target_properties(doge-qt PROPERTIES
 		MACOSX_BUNDLE ON
 		OUTPUT_NAME "${BITCOIN_QT_OSX_BUNDLE_NAME}"
 		MACOSX_BUNDLE_INFO_PLIST "${CMAKE_SOURCE_DIR}/share/qt/Info.plist.cmake.in"
 		MACOSX_BUNDLE_BUNDLE_NAME "${BITCOIN_QT_OSX_BUNDLE_NAME}"
-		MACOSX_BUNDLE_BUNDLE_VERSION "${bitcoin-abc_VERSION}"
+		MACOSX_BUNDLE_BUNDLE_VERSION "${doged-node_VERSION}"
 		MACOSX_BUNDLE_GUI_IDENTIFIER "org.bitcoinabc.${BITCOIN_QT_OSX_BUNDLE_NAME}"
 		MACOSX_BUNDLE_ICON_FILE "${BITCOINQT_BUNDLE_ICON_NAME}"
-		MACOSX_BUNDLE_INFO_STRING "${bitcoin-abc_VERSION}, Copyright © 2009-${COPYRIGHT_YEAR} ${COPYRIGHT_HOLDERS_FINAL}"
-		MACOSX_BUNDLE_LONG_VERSION_STRING "${bitcoin-abc_VERSION}"
-		MACOSX_BUNDLE_SHORT_VERSION_STRING "${bitcoin-abc_VERSION}"
+		MACOSX_BUNDLE_INFO_STRING "${doged-node_VERSION}, Copyright © 2009-${COPYRIGHT_YEAR} ${COPYRIGHT_HOLDERS_FINAL}"
+		MACOSX_BUNDLE_LONG_VERSION_STRING "${doged-node_VERSION}"
+		MACOSX_BUNDLE_SHORT_VERSION_STRING "${doged-node_VERSION}"
 		RESOURCE "${BITCOINQT_BUNDLE_RESOURCES}"
 	)
 	# The InfoPlist.strings files should be located in a resource subdirectory.
diff --git a/src/qt/guiconstants.h b/src/qt/guiconstants.h
index b14f008835629cac18eb10fbc3834e2479d61702..f55a1e7c21241e5f5e9cec480e933456d9c93bb6 100644
--- a/src/qt/guiconstants.h
+++ b/src/qt/guiconstants.h
@@ -45,9 +45,9 @@ static const int TOOLTIP_WRAP_THRESHOLD = 80;
 
 #define QAPP_ORG_NAME "BitcoinABC"
 #define QAPP_ORG_DOMAIN "bitcoinabc.org"
-#define QAPP_APP_NAME_DEFAULT "BitcoinABC-Qt"
-#define QAPP_APP_NAME_TESTNET "BitcoinABC-Qt-testnet"
-#define QAPP_APP_NAME_REGTEST "BitcoinABC-Qt-regtest"
+#define QAPP_APP_NAME_DEFAULT "Doge-Qt"
+#define QAPP_APP_NAME_TESTNET "Doge-Qt-testnet"
+#define QAPP_APP_NAME_REGTEST "Doge-Qt-regtest"
 
 /* One gigabyte (GB) in bytes */
 static constexpr uint64_t GB_BYTES{1'000'000'000};
